js/data/bucket.js

```diff
@@ -223,7 +225,7 @@ Bucket.prototype.setAttribPointers = function(programName, gl, program, offset,
     var disabledAttributes = this.attributes[programName].disabled;
     for (var i = 0; i < disabledAttributes.length; i++) {
         attribute = disabledAttributes[i];
-        if (attribute.isLayerConstant === false && layer.id !== attribute.layerId) continue;
+        assert(attribute.paintProperty);
```

Comment:
> it's safe to remove the attribute.isLayerConstant === false because this is always true for disabled attributes, right?

Correct (as of the new `paintProperty` implementation) 👍